server/sonar-server/src/main/java/org/sonar/server/permission/index/PermissionIndexer.java

```diff
@@ -77,9 +78,11 @@ public PermissionIndexer(DbClient dbClient, EsClient esClient) {
   public void indexAllIfEmpty() {
     Future submit = executor.submit(() -> {
       if (isIndexEmpty(IssueIndexDefinition.INDEX, IssueIndexDefinition.TYPE_AUTHORIZATION) ||
-        isIndexEmpty(ProjectMeasuresIndexDefinition.TYPE_PROJECT_MEASURES, ProjectMeasuresIndexDefinition.TYPE_AUTHORIZATION)) {
+        isIndexEmpty(ProjectMeasuresIndexDefinition.TYPE_PROJECT_MEASURES, ProjectMeasuresIndexDefinition.TYPE_AUTHORIZATION) ||
```

Comment:
As discussed, a future refactoring will aim to replacing this hardcoded list by an interface shared by the indices requiring authorization.